extension/intents/music/music.js

```diff
@@ -94,6 +94,10 @@ intentRunner.registerIntent({
 intentRunner.registerIntent({
   name: "music.unpause",
   async run(context) {
+    for (const ServiceClass of Object.values(SERVICES)) {
+      const service = new ServiceClass(context);
+      await service.pauseAny();
```

Comment:
Right here you could use `service.pauseAny({exceptTabId: (await browserUtils.activeTab()).id)` and then if the current tab is already playing it won't get paused and unpaused.

Reply:
Requested changes have been made.